Scripts/script-CommonServerPython.yml

```diff
@@ -1079,6 +1079,39 @@ script: |-
   sha256Regex = re.compile(r'\b[0-9a-fA-F]{64}\b', regexFlags)
 
   ############################### REGEX FORMATTING end ###############################
+
+  def key_to_table_key(string):
+      """
+        Checks if string, change underscores to spaces, capitalize every word.
+        Example: "one_two" to "One Two"
+
+        :type string: ``str``
+        :param string: The string to be converted (required)
+
+        :return: The converted string
+        :rtype: ``string``
+      """
+      if isinstance(string, basestring):
+          return " ".join(word.capitalize() for word in string.replace("_"," ").split())
+      else:
+          return_error('The key is not a string: {}'.format(string))
+
+  def key_to_context_key(string):
```

Comment:
I recommend to rename it to `key_to_upper_camelcase`

Reply:
Technically its more like key_to_title except the spacing part.